src/main/java/net/dries007/tfc/objects/items/rock/ItemRockAxe.java

```diff
@@ -47,9 +48,9 @@ public ItemRockAxe(RockCategory category)
         this.category = category;
         if (MAP.put(category, this) != null) throw new IllegalStateException("There can only be one.");
         setHarvestLevel("axe", category.getToolMaterial().getHarvestLevel());
-        OreDictionaryHelper.register(this, "axe");
-        OreDictionaryHelper.register(this, "axe", "stone");
-        OreDictionaryHelper.register(this, "axe", "stone", category);
+        OreDictionaryHelper.registerMeta(this, OreDictionary.WILDCARD_VALUE, "axe");
```

Comment:
Why do tools even need ore dictionary values beyond `axe`? Also see above re: `registerMeta` should be replaced with `register`